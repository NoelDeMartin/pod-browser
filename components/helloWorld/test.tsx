import React from 'react';
import { shallow } from 'enzyme';

import HelloWorld from './index';


test('renders a container with some typography and a button', () => {
  const wrap = shallow(<HelloWorld />);
  expect(wrap).toMatchSnapshot();
});
