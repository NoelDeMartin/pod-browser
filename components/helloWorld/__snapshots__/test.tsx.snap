// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a container with some typography and a button 1`] = `ShallowWrapper {}`;
